### PR TITLE
Add escaping to make helm work

### PR DESCRIPTION
**Pull request by MarcelMue**

### CODE REVIEW

**Comment by MarcelMue** [comment]
@stone-z What is the best way to test the helm stuff? Or how did you test it before?

**Comment by stone-z** [comment]
Either `ct lint` it or just try to install with helm. Also [related upstream issue](https://github.com/kyverno/kyverno/issues/1751) FYI

File: template.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -euo pipefail
+
+for i in aws azure common vmware; do
+  mkdir -p helm/policies-$i/templates
+
+  cp -a policies/$i/. helm/policies-$i/templates
+
+  for filename in helm/policies-$i/templates/*.yaml; do
+    sed -i 's/{{/üüü/g' $filename # replace {{ with placeholder
```
Comment (stone-z) [inline template.sh:10]: :laughing:

**Comment by MarcelMue** [comment]
> Either `ct lint` it or just try to install with helm. Also [related upstream issue](https://github.com/kyverno/kyverno/issues/1751) FYI

I tried it with `helm package` and then `helm install` which for me locally worked fine on kind.

**Review by stone-z — APPROVED** [review]

**Pull request merged**